doc/classes/VisualServer.xml

```diff
@@ -1402,6 +1527,7 @@
 			<argument index="0" name="texture" type="RID">
 			</argument>
 			<description>
+				Returns the height of an texture's image.
```

Comment:
This and the ones below: "an texture" -> "the texture".  Definite article is preferred over indefinite.

Also, "texture's image" is redundant. Why not just "the texture's height"?